setup.bat

```diff
@@ -0,0 +1 @@
+mvn package
```

Comment:
If you're adding Windows scripts, it should include equivalents to all of our scripts, and be in a separate PR.